src/main/resources/script_templates/ImageJ2/Manual_Registration.py

```diff
@@ -0,0 +1,76 @@
+# @ImageJ ij
+# @Dataset ds
+# @OpService ops
+# @DatasetService datasetService
+# @OUTPUT Dataset final_dataset
+
+import math
+
+from net.imglib2.util import Intervals
+from net.imagej.axis import Axes
+from net.imglib2.view import Views
+
+from ij.plugin.frame import RoiManager
+
+# Initialize some variables
+img = ds.getImgPlus()
+rois = RoiManager.getInstance().getRoisAsArray()
+
+images = []
+
+total_x_shift = 0
+total_y_shift = 0
+
+# Iterate over regions of interest
+for j, r in enumerate(range(len(rois)-1)):
+
+	# Get two ROIs
+	start = rois[r]
+	end = rois[r+1]
+
+	# Get Z or T positions defined by the position of the ROIs
+	z_start = start.getPosition()
+	z_end = end.getPosition()
+
+	# Get X positions
+	x_start = start.getContainedPoints()[0].x
+	x_end = end.getContainedPoints()[0].x
+
+	# Get Y positions
+	y_start = start.getContainedPoints()[0].y
+	y_end = end.getContainedPoints()[0].y
+
+	# Calculate the linear translation for each frame in the Z/T axis
+	x_shift = float(x_end - x_start) / (z_end - z_start)
+	y_shift = float(y_end - y_start) / (z_end - z_start)
+
+	# Iterate over each frame in Z/T
+	for i, z in enumerate(range(z_start, z_end)):
+
+		ij.log().info("Processing frame %i/%i for roi #%i" % (i + 1, z_end - z_start, j))
```

Comment:
This seems to be the only place where you are using the `ImageJ` gateway. Can't we replace the very first line of the script with `@LogService log` and this line with `log.info(...)`?